### PR TITLE
feat added: simple desktop navbar

### PR DESCRIPTION
**Pull request by rdshinde**
Added stylesheet for simple desktop navbar.
Screenshot 2022-02-05 at 10 54 06 PM
Screenshot 2022-02-05 at 10 54 44 PM

### CODE REVIEW

**Review by SimranjitKaurMaan — COMMENTED** [review]
lgtm

File: components/navigation/navbar.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* Styling for navbar */
```
Comment (ImrozNezami) [inline components/navigation/navbar.css:1]: remove comment

File: components/navigation/navbar.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  align-self: flex-end;
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: auto;
+}
+.nav ul li {
+  margin: 0 var(--scale-sm);
+}
+
+/* Styling For Dark Navbar */
```
Comment (ImrozNezami) [inline components/navigation/navbar.css:33]: remove comment

File: components/navigation/navbar.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+.nav__bg-dark {
+  background-color: var(--dark-color);
+}
+.nav__bg-dark a,
+* {
+  color: var(--off-white);
+}
+
+/* Adding Breakpoints */
```
Comment (ImrozNezami) [inline components/navigation/navbar.css:42]: remove comment

**Pull request merged**